units/crons.go

```diff
@@ -685,6 +686,36 @@ func PopulateAgentDeployJobs(env evergreen.Environment) amboy.QueueOperation {
 
 }
 
+// PopulateGenerateTasksJobs poulates generate.tasks jobs for tasks that have started running their generate.tasks command.
+func PopulateGenerateTasksJobs(env evergreen.Environment) amboy.QueueOperation {
+	return func(ctx context.Context, _ amboy.Queue) error {
```

Comment:
this is awful and we should have a periodic queueGroup operation, can you open an MAKE ticket for periodic operations with queue groups? 

I'd almost rather just write a function that takes a group and run it seperatly in the cron

Reply:
https://jira.mongodb.org/browse/MAKE-932